crates/rerun_c/src/rerun.hpp

```diff
@@ -0,0 +1,76 @@
+// The Rerun C++ SDK.
+
+#ifndef RERUN_HPP
+#define RERUN_HPP
+
+#include <rerun.h>
+
+namespace rerun {
+    inline const char* version_string() {
+        return rerun_version_string();
+    }
+} // namespace rerun
+
+// ----------------------------------------------------------------------------
+// Arrow integration
+
+#if RERUN_WITH_ARROW
+
+#include <arrow/api.h>
+#include <arrow/io/api.h>
+#include <arrow/ipc/api.h>
+
+#include <loguru.hpp>
+
+namespace rerun {
+    arrow::Result<std::shared_ptr<arrow::Table>> points3(size_t num_points,
```

Comment:
Nice! This is actually quite similar to the Rust Arrow SDK, it should be pretty straightforward to codegen starting with what we already have for Rust.